addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -747,6 +747,28 @@ test("automatically cancel incoming call after some time", async () => {
     await contains(".o-discuss-CallInvitation", { count: 0 });
 });
 
+test("Should not auto-cancel incoming call when camera preview is open", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    const [memberId] = pyEnv["discuss.channel.member"].search([["channel_id", "=", channelId]]);
+    const rtcSessionId = pyEnv["discuss.channel.rtc.session"].create({
+        channel_member_id: memberId,
+        channel_id: channelId,
+    });
+    pyEnv["discuss.channel.member"].write([memberId], { rtc_inviting_session_id: rtcSessionId });
+    await start();
+    await openDiscuss(channelId);
+    await contains(".o-discuss-CallInvitation");
+    await advanceTime(25_000);
+    await contains(".o-discuss-CallInvitation");
+    await click(".o-mail-ActionList-button[title='Show camera preview']");
+    await advanceTime(25_000); // Call should not auto-cancel while preview is open
+    await contains(".o-discuss-CallInvitation");
+    await click(".o-mail-ActionList-button[title='Hide camera preview']");
+    await advanceTime(30_000); // Timer restarts when the preview closes, and the call should auto-cancel after 30s.
```

Comment:
Could import and use `ChannelMember.CANCEL_CALL_INVITE_DELAY`, so that the test rely less on magic value.